src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -228,6 +228,9 @@ def configure_container_dict(
         host_mount_path = volumes_string.replace(":" + container_mount_path, "")
         logger.debug(f"host_mount_path: {host_mount_path}")
         logger.debug(f"container_mount_path: {container_mount_path}")
+    logger.info(
+        f"PyFluent workdir: Fluent workdir {host_mount_path}:{container_mount_path}"
+    )
```

Comment:
I think this would be more appropriate as a warning so that it is more visible to users, and more clear and detailed so as to alleviate confusion that may happen when using Fluent containers:
```suggestion
    logger.warning(
        f"Starting Fluent container mounted to {host_mount_path}, with this path available as {container_mount_path} for the Fluent session running inside the container."
    )
```

Reply:
Done. Thank you.